cmake/Projects.cmake

```
@@ -142,6 +142,21 @@ foreach(project ${LY_PROJECTS})
     add_subdirectory(${project} "${project_folder_name}-${full_directory_hash}")
     ly_generate_project_build_path_setreg(${full_directory_path})
     add_project_json_external_subdirectories(${full_directory_path})
+
+    # Generate pak for project in release installs
+    cmake_path(RELATIVE_PATH CMAKE_RUNTIME_OUTPUT_DIRECTORY BASE_DIRECTORY ${CMAKE_BINARY_DIR} OUTPUT_VARIABLE runtime_output_directory)
+    ly_install_run_code("
+if(\"\${CMAKE_INSTALL_CONFIG_NAME}\" MATCHES \"^([Rr][Ee][Ll][Ee][Aa][Ss][Ee])$\")
+    set(install_output_folder \"\${CMAKE_INSTALL_PREFIX}/${runtime_output_directory}/${PAL_PLATFORM_NAME}/\${CMAKE_INSTALL_CONFIG_NAME}\")
+    message(STATUS \"Generating \${install_output_folder}/Engine.pak from ${full_directory_path}/Cache\")
+    file(ARCHIVE_CREATE OUTPUT \${install_output_folder}/Engine.pak
+        PATHS ${full_directory_path}/Cache
+        FORMAT zip
+    )
+    message(STATUS \"\${install_output_folder}/Engine.pak generated\")
+endif()
+")
```

The reduce the number of backslash escapes, we can use a combination of [bracket arguments](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html?highlight=bracket%20arguments#bracket-argument) and [string(CONFIGURE)](https://cmake.org/cmake/help/latest/command/string.html#configure).

Now this is just a optional ask and can be ignored, if you don't think it helps readability
```suggestion
set(install_engine_pak_template [=[if("${CMAKE_INSTALL_CONFIG_NAME}" MATCHES "^([Rr][Ee][Ll][Ee][Aa][Ss][Ee])$")
    set(install_output_folder "${CMAKE_INSTALL_PREFIX}/@runtime_output_directory@/@PAL_PLATFORM_NAME@/${CMAKE_INSTALL_CONFIG_NAME}")
    message(STATUS "Generating ${install_output_folder}/Engine.pak from @full_directory_path@/Cache")
    file(ARCHIVE_CREATE OUTPUT ${install_output_folder}/Engine.pak
        PATHS @full_directory_path@/Cache
        FORMAT zip
    )
    message(STATUS "${install_output_folder}/Engine.pak generated")
endif()
]=])
string(CONFIGURE "${install_engine_pak_template}" install_engine_pak_code @ONLY)
ly_install_run_code("${install_engine_pak_code}")
```

---

I like it, taking it but changing it slightly to make the first `if` on line 148 to start in a new line